### PR TITLE
Add primary contributors to Acknowledgements

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

**Comment by ethanwhite** [comment]
Let me know if there's anything you'd like to see changed in this @brymz.

File: about/acknowledgements.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ layout: page
 title: Acknowledgments
 ---
 
+#### Primary Contributors
+
+This material, including lectures, website design and implementation, and
```
Comment (brymz) [inline about/acknowledgements.md]: How about just: "Course materials and website design have been primarily developed and implemented by"?

**Comment by brymz** [comment]
Thanks, @ethanwhite, for the acknowledgment!

**Comment by ethanwhite** [comment]
Yeah, that's better. Changed to your version.

**Pull request merged**